.github/workflows/R-CMD-check.yaml

```diff
@@ -148,10 +148,10 @@ jobs:
               if (is_valid_os(mac, rver$release))                             job(mac, rver$release),
               if (has_src && is_valid_os(windows, "devel"))                   job(windows, "devel"),
               if (is_valid_os(windows, rver$release))                         job(windows, rver$release),
+              if (has_src && is_valid_os(windows, "4.4"))                     job(windows, "4.4", "rtools-version" = "44"),
               if (has_src && is_valid_os(windows, "4.3"))                     job(windows, "4.3", "rtools-version" = "43"),
               if (has_src && is_valid_os(windows, "4.2"))                     job(windows, "4.2", "rtools-version" = "42"),
               if (has_src && test_on_rtools40 && is_valid_os(windows, "4.1")) job(windows, "4.1", "rtools-version" = "40"),
-              if (has_src && test_on_rtools35 && is_valid_os(windows, "3.6")) job(windows, "3.6"),
```

Comment:
Maybe we should also remove the `rtools_35` input as well? Or at least update the documentation to say "ignored".

Reply:
I just removed the stuff about rtools 35, since I think it would only have been tested on R 3.6?